### ISSUE TITLE
Add a center prop to geometry?

### ISSUE THREAD

**Issue by DelvarWorld** [toxicFork/react-three-renderer#38]
By default, text geometry isn't centered. So wherever you place the text, it will start leftmost aligned at that position.

The Three.js examples get around this by using [`geoemtry.center()`](http://threejs.org/docs/#Reference/Core/Geometry.center). This might be a nice prop to add to geometry, which bases the center point on the bounding box of the geometry.

**Comment by toxicFork** [comment]
Yes :) thinking of calling this every time the geometry updates if a bool prop is true. Comments welcome.
